users/models.py

```diff
@@ -32,6 +32,28 @@ def create_user(self, email, username, password):
 
         return user
 
+    def create_superuser(self, email, username, password):
+        if not email:
+            raise ValueError('Superusers must have an email address')
+
+        if not username:
+            raise ValueError('Superusers must have an username')
+
+        if not password:
+            raise ValueError('Superusers must have a password')
+
+        superuser = self.model(
+            email=self.normalize_email(email),
```

Comment:
I copied that code from you, so it must be the format we use. Ha! :D
Also, according to PEP you do not use spaces around `=`s in kwargs.